operators/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "operators"
+version = "0.1.0"
+authors = ["Tiago Castro <tiago.castro@mayadata.io>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[[bin]]
+name = "node-op"
+path = "node/src/main.rs"
+
+[dependencies]
+actix-web = "3.2.0"
+serde_json = "1.0"
+structopt = "0.3.15"
+log = "0.4.11"
+env_logger = "0.7"
```

Comment:
I wonder if we should not use tracing here from the start? :thinking:

Reply:
yeah we probably should, I'll have a chat with @cjones1024